Source/WebCore/rendering/RenderElement.h

```diff
@@ -25,6 +25,7 @@
 #include "HitTestRequest.h"
 #include "LengthFunctions.h"
 #include "RenderObject.h"
+#include "RenderStyle.h"
```

Comment:
No way to avoid this one? I think you meant to do this only in the `Inlines.h` file.

Reply:
Unfortunately no. RenderElement has a `RenderStyle m_style;` ivar.  Removing `#include "RenderStyle.h"` from RenderObject.h means it must be included here.

This could be worked around by making it a `UniqueRef<RenderStyle> m_style;` and moving a bunch of methods to the inlines header, but that opens up a thorny issue about whether increasing the size of a RenderElement by the size of a UniqueRef and possibly adding an indirection on all style lookups is worth the build speed reduction. So I erred on the side of the status quo.

Reply:
You should probably remove the include from the `RenderStyleInlines.h` header.